### PR TITLE
fix runtime deps by adding coreutils

### PR DESCRIPTION
**Pull request by developer-guy**
thx @rawlingsj for figuring this out!

### CODE REVIEW

**Comment by rawlingsj** [comment]
I've tested this locally and seems to work.

**Review by rawlingsj — DISMISSED** [review]

**Comment by marcofranssen** [comment]
Resolves https://github.com/chainguard-images/images/issues/136

**Review by marcofranssen — DISMISSED** [review]

**Comment by marcofranssen** [comment]
Ready to test another release https://github.com/philips-labs/helm-charts/pull/80/files

**Comment by rawlingsj** [comment]
Looks like the script needs busybox here https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh#L144-L149

**Comment by rawlingsj** [comment]
@developer-guy so it turns out we can just use `coreutils` instead of adding `busybox` would you mind changing this PR and we'll get it merged in

File: wait-for-it.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ package:
   dependencies:
     runtime:
       - bash
+      - busybox
```
Comment (rawlingsj) [inline wait-for-it.yaml:16]: this can be `coreutils` which avoids busybox

**Review by amouat — APPROVED** [review]

**Comment by developer-guy** [comment]
Can we merge this one, @kaniini, because it will fix the wait-for-it OCI image? We have a bug right now 😩

**Review by kaniini — APPROVED** [review]

**Pull request merged**